eng/pipelines/runtime.yml

```diff
@@ -1078,7 +1078,10 @@ jobs:
       liveRuntimeBuildConfig: release
       runtimeVariant: llvmaot
       condition: >-
-        eq(variables['isFullMatrix'], true)
+        or(
+          eq(dependencies.evaluate_paths.outputs['SetPathVars_libraries.containsChange'], true),
```

Comment:
Is there enough value in running runtime tests for libraries changes? The runtime tests should not be testing much of the libraries.

Reply:
No, looks like copy/paste mistake from a long time ago to me.

Reply:
done